backend/config/config_schema.json

```diff
@@ -0,0 +1,178 @@
+{
+  "type": "object",
+  "properties": {
+    "docker": {
+      "type": "object",
+      "properties": {
+        "registry": {
+          "type": "string"
+        }
+      }
+    },
+    "operators": {
+      "type": "object",
+      "deprecated": true,
```

Comment:
Should `operators` be deprecated?

Reply:
I marked it as deprecated since #440 removes the unnecessary abstraction since we no longer use Docker.

Reply:
But it's also not supported anymore, right? Deprecated means for me: Still works, but please change it. When it doesn't work anymore, we should not accept it.